src/AdminConsole/Services/PasswordlessManagement/PasswordlessManagementClient.cs

```diff
@@ -152,14 +152,14 @@ public async Task DisabledManuallyGeneratedTokensAsync(string appId, string perf
         response.EnsureSuccessStatusCode();
     }
 
-    public async Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request) =>
-        await http.GetFromJsonAsync<GetAppIdAvailabilityResponse>(
+    public Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request) =>
```

Comment:
Why remove `async`/`await` here? I generally prefer explicit awaits so that the exception contains the full stacktrace

Reply:
I was choosing between this
```
    public Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request) =>
        http.GetFromJsonAsync<GetAppIdAvailabilityResponse>(
            $"admin/apps/{Uri.EscapeDataString(request.AppId)}/available"
        )!;
 ```
 
 And this
 ```
    public async Task<GetAppIdAvailabilityResponse> IsApplicationIdAvailableAsync(GetAppIdAvailabilityRequest request) =>
        (await http.GetFromJsonAsync<GetAppIdAvailabilityResponse>(
            $"admin/apps/{Uri.EscapeDataString(request.AppId)}/available"
        ))!;
```

And opted not to have that extra set of parentheses.  I can change it back though if that's preferred.

Reply:
Personally I'd vote for extra parens here